From 4e894d5553392d0891aad32253ff4cca757e1ab6 Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Fri, 11 Apr 2014 18:02:06 -0700
Subject: [PATCH 3932/5965] Update M8 default kernel config.

Sync up between TrustZone enabled kernel config and normal kernel config.
---
 arch/arm/configs/meson8_defconfig     | 33 ++++++++-------------------
 arch/arm/configs/meson8_tee_defconfig | 23 +++++++++++--------
 2 files changed, 23 insertions(+), 33 deletions(-)
 mode change 100755 => 100644 arch/arm/configs/meson8_defconfig

diff --git a/arch/arm/configs/meson8_defconfig b/arch/arm/configs/meson8_defconfig
old mode 100755
new mode 100644
index 708e0157397c..300f400ce10b
--- a/arch/arm/configs/meson8_defconfig
+++ b/arch/arm/configs/meson8_defconfig
@@ -204,10 +204,10 @@ CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AML_NAND_KEY=y
 # CONFIG_AM_NAND is not set
 CONFIG_AM_ETHERNET=y
-CONFIG_AML_IP101_PHY=y
 CONFIG_AML_PHY=y
-CONFIG_AML_KSZ8091=y
 CONFIG_AML_LAN8720=y
+CONFIG_AML_IP101_PHY=y
+CONFIG_AML_KSZ8091=y
 CONFIG_AM_WIFI=y
 CONFIG_AM_WIFI_SD_MMC=y
 CONFIG_AM_WIFI_USB=y
@@ -222,7 +222,6 @@ CONFIG_AM_ENCODER=y
 CONFIG_AM_JPEG_ENCODER=y
 CONFIG_AM_TV_OUTPUT2=y
 CONFIG_FB_AM=y
-CONFIG_FB_OSD2_CURSOR=n
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT=32
 CONFIG_FB_OSD2_DEFAULT_WIDTH_VIRTUAL=32
@@ -269,11 +268,13 @@ CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_DEV_SR=y
 CONFIG_SCSI_MULTI_LUN=y
+CONFIG_MD=y
+CONFIG_BLK_DEV_DM=y
+CONFIG_DM_DEBUG=y
+CONFIG_DM_CRYPT=y
+CONFIG_DM_UEVENT=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
-#CONFIG_MICREL_PHY=y
-#CONFIG_SMSC_PHY=y
-#CONFIG_ICPLUS_PHY=y
 CONFIG_PPP=y
 CONFIG_PPP_BSDCOMP=y
 CONFIG_PPP_DEFLATE=y
@@ -308,14 +309,12 @@ CONFIG_USB_SERIAL_OPTION=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_GADGET_DWC_OTG=y
 CONFIG_USB_G_ANDROID=y
-CONFIG_MMC_DEBUG=n
 CONFIG_UIO=y
 CONFIG_UIO_PDRV=y
 CONFIG_UIO_PDRV_GENIRQ=y
 CONFIG_STAGING=y
 CONFIG_ZSMALLOC=y
 CONFIG_ZRAM=y
-CONFIG_ZRAM_DEBUG=n
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_ASHMEM=y
@@ -330,7 +329,6 @@ CONFIG_EXT3_FS_SECURITY=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y
 CONFIG_EXT4_FS_SECURITY=y
-CONFIG_EXT4_DEBUG=n
 CONFIG_QUOTA=y
 CONFIG_QFMT_V2=y
 CONFIG_FUSE_FS=y
@@ -350,26 +348,13 @@ CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_READABLE_ASM=y
 CONFIG_DEBUG_FS=y
-CONFIG_DEBUG_SECTION_MISMATCH=n
 CONFIG_SCHEDSTATS=y
 CONFIG_TIMER_STATS=y
 CONFIG_DEBUG_INFO=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
 CONFIG_SECURITY_SELINUX=y
-CONFIG_CRC_T10DIF=y
-CONFIG_CRC7=y
-CONFIG_MD=y
-CONFIG_BLK_DEV_DM=y
-CONFIG_DM_DEBUG=y
-CONFIG_DM_CRYPT=y
-CONFIG_DM_UEVENT=y
-CONFIG_CRYPTO_AEAD=y
-CONFIG_CRYPTO_AUTHENC=y
-CONFIG_CRYPTO_CBC=y
 CONFIG_CRYPTO_PCBC=y
-CONFIG_CRYPTO_HMAC=y
-CONFIG_CRYPTO_MD5=y
-CONFIG_CRYPTO_DES=y
 CONFIG_CRYPTO_TWOFISH=y
-CONFIG_CRYPTO_TWOFISH_COMMON=y
+CONFIG_CRC_T10DIF=y
+CONFIG_CRC7=y
diff --git a/arch/arm/configs/meson8_tee_defconfig b/arch/arm/configs/meson8_tee_defconfig
index ef05b9147d3d..5ab9bb34d1e9 100644
--- a/arch/arm/configs/meson8_tee_defconfig
+++ b/arch/arm/configs/meson8_tee_defconfig
@@ -17,6 +17,7 @@ CONFIG_MODULES=y
 CONFIG_MODULE_UNLOAD=y
 CONFIG_MESON_SUSPEND=y
 CONFIG_MESON_LEGACY_REGISTER_API=y
+CONFIG_MESON_TRUSTZONE=y
 CONFIG_ARM_ERRATA_751472=y
 CONFIG_ARM_ERRATA_754322=y
 CONFIG_ARM_ERRATA_764369=y
@@ -24,7 +25,6 @@ CONFIG_SMP=y
 CONFIG_PREEMPT=y
 CONFIG_AEABI=y
 CONFIG_HIGHMEM=y
-CONFIG_KSM=y
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_INTERACTIVE=y
@@ -183,6 +183,7 @@ CONFIG_GRAVITY_MM3A310=y
 CONFIG_GRAVITY_MXC622X=y
 CONFIG_LIGHT_CM36283=y
 CONFIG_LIGHT_CM3232=y
+CONFIG_LIGHT_CM3217=y
 CONFIG_LIGHT_LTR501=y
 CONFIG_LIGHT_EPL6814=y
 CONFIG_LIGHT_ISL29023=y
@@ -192,6 +193,7 @@ CONFIG_AML_GPIO_KEY=y
 CONFIG_AML_PMU_ALGORITHM_SUPPORT=m
 CONFIG_AML_PMU=y
 CONFIG_AML1216=y
+CONFIG_AML1218=y
 CONFIG_RICOH_PMU=y
 CONFIG_SECURITYKEY=y
 CONFIG_EFUSE=y
@@ -203,6 +205,10 @@ CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AML_NAND_KEY=y
 # CONFIG_AM_NAND is not set
 CONFIG_AM_ETHERNET=y
+CONFIG_AML_PHY=y
+CONFIG_AML_LAN8720=y
+CONFIG_AML_IP101_PHY=y
+CONFIG_AML_KSZ8091=y
 CONFIG_AM_WIFI=y
 CONFIG_AM_WIFI_SD_MMC=y
 CONFIG_AM_WIFI_USB=y
@@ -217,7 +223,6 @@ CONFIG_AM_ENCODER=y
 CONFIG_AM_JPEG_ENCODER=y
 CONFIG_AM_TV_OUTPUT2=y
 CONFIG_FB_AM=y
-CONFIG_FB_OSD2_CURSOR=y
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT=32
 CONFIG_FB_OSD2_DEFAULT_WIDTH_VIRTUAL=32
@@ -253,7 +258,6 @@ CONFIG_V4L_AMLOGIC_VIDEO=y
 CONFIG_V4L_AMLOGIC_VIDEO2=y
 CONFIG_AMLOGIC_IONVIDEO=y
 CONFIG_DEINTERLACE=y
-CONFIG_MESON_TRUSTZONE=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_CMA=y
@@ -265,10 +269,13 @@ CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_DEV_SR=y
 CONFIG_SCSI_MULTI_LUN=y
+CONFIG_MD=y
+CONFIG_BLK_DEV_DM=y
+CONFIG_DM_DEBUG=y
+CONFIG_DM_CRYPT=y
+CONFIG_DM_UEVENT=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
-CONFIG_SMSC_PHY=y
-CONFIG_ICPLUS_PHY=y
 CONFIG_PPP=y
 CONFIG_PPP_BSDCOMP=y
 CONFIG_PPP_DEFLATE=y
@@ -303,14 +310,12 @@ CONFIG_USB_SERIAL_OPTION=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_GADGET_DWC_OTG=y
 CONFIG_USB_G_ANDROID=y
-CONFIG_MMC_DEBUG=y
 CONFIG_UIO=y
 CONFIG_UIO_PDRV=y
 CONFIG_UIO_PDRV_GENIRQ=y
 CONFIG_STAGING=y
 CONFIG_ZSMALLOC=y
 CONFIG_ZRAM=y
-CONFIG_ZRAM_DEBUG=y
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_ASHMEM=y
@@ -325,7 +330,6 @@ CONFIG_EXT3_FS_SECURITY=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y
 CONFIG_EXT4_FS_SECURITY=y
-CONFIG_EXT4_DEBUG=y
 CONFIG_QUOTA=y
 CONFIG_QFMT_V2=y
 CONFIG_FUSE_FS=y
@@ -345,12 +349,13 @@ CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_READABLE_ASM=y
 CONFIG_DEBUG_FS=y
-CONFIG_DEBUG_SECTION_MISMATCH=y
 CONFIG_SCHEDSTATS=y
 CONFIG_TIMER_STATS=y
 CONFIG_DEBUG_INFO=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
 CONFIG_SECURITY_SELINUX=y
+CONFIG_CRYPTO_PCBC=y
+CONFIG_CRYPTO_TWOFISH=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
-- 
2.19.0

